server.js

```diff
@@ -113,8 +113,12 @@ var helmetOptions = {
 app.use('/assets', express.static('assets', {
 	maxAge: '31536000000' 
 }));
-app.use(cors(corsOptions));
-app.use(helmet(helmetOptions));
+if (process.env.ALLOW_HTTP==="") {
```

Comment:
I'll admit I'm somewhat un-initiate with the default behavior of these environment variables so maybe checking for empty string is enough, is it possible the value could come back as undefined/null though? If so maybe using an `isEmpty()` function would be safer..